pom.xml

```diff
@@ -19,14 +19,14 @@
 
     <dependencies>
         <dependency>
-            <groupId>org.hibernate</groupId>
+            <groupId>org.hibernate.orm</groupId>
             <artifactId>hibernate-core</artifactId>
-            <version>6.2.7.Final</version>
+            <version>6.3.1.Final</version>
         </dependency>
         <dependency>
-            <groupId>mysql</groupId>
-            <artifactId>mysql-connector-java</artifactId>
-            <version>8.0.33</version>
+            <groupId>com.mysql</groupId>
+            <artifactId>mysql-connector-j</artifactId>
+            <version>8.3.0</version>
```

Comment:
What is the reason for this change?

Reply:
there is no point for this task, but in previous ones I had problems with mokito due to out-of-date driver versions, so I got used to changing it to the newest ones. (returned to original parameters)